ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/PartyLedgerEntry.scala

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.platform.sandbox.stores.ledger
+
+import java.time.Instant
+
+import com.daml.ledger.participant.state.v1.{ParticipantId, SubmissionId}
+import com.digitalasset.ledger.api.domain.PartyDetails
+
+sealed abstract class PartyLedgerEntry() extends Product with Serializable {
+  val submissionIdOpt: Option[SubmissionId]
+  val recordTime: Instant
+}
+
+object PartyLedgerEntry {
+
+  final case class AllocationAccepted(
+      submissionIdOpt: Option[SubmissionId],
+      participantId: ParticipantId,
+      override val recordTime: Instant,
```

Comment:
```suggestion
      recordTime: Instant,
```
This should not be necessary IIRC.

Reply:
Addressed in ba16667d1.